backend/routes/resetPassword.js

```diff
@@ -14,15 +14,20 @@ function getRandomInt(max) {
 }
 
 router.post("/", (req, res) => {
+  randomNumbers = [];
+  setTimeout(() => {
+   randomNumbers = [];
+  }, 900000);
```

Comment:
?

Reply:
validation 4 digits code is deleted (empty array) after timer ends